bin/lib/absorb.js

```diff
@@ -323,6 +326,20 @@ function getDataFromURL(feedInfo){
 		})
 	;
 
+	const currentTime = new Date();
+	
+	// If it's between 13:00 and 13:10, and we have not sent a problem report
```

Comment:
It's just as I get back from lunch, and we usually have content from 3rd parties by then, so we can act on errors quickly.